<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vector</class>
 <widget class="QWidget" name="Vector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>35</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="labelX">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="boxX">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelY">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="boxY">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelZ">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="boxZ">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boxX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Vector</receiver>
   <slot>changeX(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Vector</receiver>
   <slot>changeY(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Vector</receiver>
   <slot>changeZ(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>valueChanged()</signal>
  <slot>changeX(double)</slot>
  <slot>changeY(double)</slot>
  <slot>changeZ(double)</slot>
 </slots>
</ui>
